### ISSUE TITLE
Textdisplays not visible when joining with 1.20.1 on 1.20.2 Server

### ISSUE THREAD

**Issue by MELONEODERSO** [ViaVersion/ViaBackwards#599]
### '/viaversion dump' Output

https://dump.viaversion.com/967ff66790e63a0e91ed37f15495b567da9ac579655c8b11b6146ecb83a6544a

### Console Error

```
Put the paste.gg link or text here.
```

### Bug Description

Textdisplays are not visible when joining with client version 1.20.1 on server version 1.20.2. 1.20.2 clients can see the Textdisplays tho. 

### Steps to Reproduce

1. Spawn Textdisplay on 1.20.2 Server
2. Join with 1.20.1 client version
3. Can't see the Textdisplays

### Expected Behavior

1. Spawn Textdisplay on 1.20.2 Server
2. Join with 1.20.1 client version
3. Textdisplays are visible 

### Additional Server Info

![Bild_2023-10-06_223029052](https://github.com/ViaVersion/ViaBackwards/assets/82024032/789673e8-45f1-4b06-b7fb-92967a7cf6e0)

### Checklist

- [X] Via plugins are only running on **EITHER** the backend servers (e.g. Paper) **OR** the proxy (e.g. BungeeCord), **not on both**.
- [X] I have included a ViaVersion dump.
- [X] If applicable, I have included a paste (**not a screenshot**) of the error.
- [X] I have tried a build from https://ci.viaversion.com/ and the issue still persists.

**Comment by Barvalg** [comment]
![Platform: git--Purpur--2074%20%28MC%3A%201.20.2%29](https://img.shields.io/badge/Platform-git--Purpur--2074%20%28MC%3A%201.20.2%29-3498db?style=flat-square) </br>![ViaVersion (4.8.1): Even with master](https://img.shields.io/badge/ViaVersion%20(4.8.1)-Even%20with%20master-brightgreen?style=flat-square) </br>![ViaBackwards(4.8.1): Even with master](https://img.shields.io/badge/ViaBackwards%20(4.8.1)-Even%20with%20master-brightgreen?style=flat-square) </br>

**Comment by Black-Hole** [comment]
This issue was reported to ViaVersion, too:
https://github.com/ViaVersion/ViaVersion/issues/3467

This also applies to item displays and block displays. The client will log errors with entity metadata:
```
[20:10:53] [Render thread/ERROR]: Error executing task on Client
java.lang.IllegalStateException: Invalid entity data item type for field 26 on entity l['Textdarsteller'/853, l='ClientLevel', x=13.99, y=79.90, z=-13.50]: old=-1(class java.lang.Byte), new=0(class java.lang.Integer)
	at aeh.a(SourceFile:204) ~[minecraft-1.20.2-client.jar:?]
	at aeh.a(SourceFile:191) ~[minecraft-1.20.2-client.jar:?]
	at fiy.a(SourceFile:516) ~[minecraft-1.20.2-client.jar:?]
	at zx.a(SourceFile:46) ~[minecraft-1.20.2-client.jar:?]
	at zx.a(SourceFile:10) ~[minecraft-1.20.2-client.jar:?]
	at fiy.a(SourceFile:2120) ~[minecraft-1.20.2-client.jar:?]
	at xh.a(SourceFile:13) ~[minecraft-1.20.2-client.jar:?]
	at xh.a(SourceFile:6) ~[minecraft-1.20.2-client.jar:?]
	at vf.a(SourceFile:23) ~[minecraft-1.20.2-client.jar:?]
	at bfo.d(SourceFile:156) ~[minecraft-1.20.2-client.jar:?]
	at bfs.d(SourceFile:23) ~[minecraft-1.20.2-client.jar:?]
	at bfo.x(SourceFile:130) ~[minecraft-1.20.2-client.jar:?]
	at bfo.bp(SourceFile:115) ~[minecraft-1.20.2-client.jar:?]
	at eqv.d(SourceFile:1231) ~[minecraft-1.20.2-client.jar:?]
	at eqv.f(SourceFile:856) ~[minecraft-1.20.2-client.jar:?]
	at net.minecraft.client.main.Main.main(SourceFile:253) ~[minecraft-1.20.2-client.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71) ~[NewLaunch.jar:?]

[20:10:53] [Render thread/ERROR]: Error executing task on Client
java.lang.IllegalStateException: Invalid entity data item type for field 13 on entity b['Blockdarsteller'/34, l='ClientLevel', x=-36.97, y=74.28, z=-0.66]: old=( 0,000E+0  0,000E+0  0,000E+0  1,000E+0)(class org.joml.Quaternionf), new=( 1,000E-1  1,000E+0  4,000E-1)(class org.joml.Vector3f)
	at aeh.a(SourceFile:204) ~[minecraft-1.20.2-client.jar:?]
	at aeh.a(SourceFile:191) ~[minecraft-1.20.2-client.jar:?]
	at fiy.a(SourceFile:516) ~[minecraft-1.20.2-client.jar:?]
	at zx.a(SourceFile:46) ~[minecraft-1.20.2-client.jar:?]
	at zx.a(SourceFile:10) ~[minecraft-1.20.2-client.jar:?]
	at fiy.a(SourceFile:2120) ~[minecraft-1.20.2-client.jar:?]
	at xh.a(SourceFile:13) ~[minecraft-1.20.2-client.jar:?]
	at xh.a(SourceFile:6) ~[minecraft-1.20.2-client.jar:?]
	at vf.a(SourceFile:23) ~[minecraft-1.20.2-client.jar:?]
	at bfo.d(SourceFile:156) ~[minecraft-1.20.2-client.jar:?]
	at bfs.d(SourceFile:23) ~[minecraft-1.20.2-client.jar:?]
	at bfo.x(SourceFile:130) ~[minecraft-1.20.2-client.jar:?]
	at bfo.bp(SourceFile:115) ~[minecraft-1.20.2-client.jar:?]
	at eqv.d(SourceFile:1231) ~[minecraft-1.20.2-client.jar:?]
	at eqv.f(SourceFile:856) ~[minecraft-1.20.2-client.jar:?]
	at net.minecraft.client.main.Main.main(SourceFile:253) ~[minecraft-1.20.2-client.jar:?]
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:88) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:126) ~[NewLaunch.jar:?]
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:71) ~[NewLaunch.jar:?]
```

1.20.2 split the "Interpolation Duration" into "Transformation interpolation duration" and "Position/Rotation interpolation duration" for display entities. So the IDs for all following entity metadata moved: https://wiki.vg/Entity_metadata#Display

**Comment by Black-Hole** [comment]
Fixed in dev branch with commit 2e0acf169b2f9f7d19ebad9d6a498af556224980
